sdk/component/target.go

```diff
@@ -0,0 +1,31 @@
+package component
+
+import (
+	"context"
+	"fmt"
+)
+
+// RunTarget runs a target after initialising the run context.
+func RunTarget(ctx context.Context, target Target, opts ...RunnerOption) error {
+	return run(
+		ctx,
+		func(ctx context.Context) error {
+			logger := LoggerFromContext(ctx).With(logKeyComponentType, "target")
+
+			logger.Debug("preparing to execute target component...")
+			logger.Debug("preparing to execute preparation step...")
+
+			if err := target.Prepare(ctx); err != nil {
+				logger.With(logKeyError, err.Error()).Error("preparing target failed")
+				return fmt.Errorf("could not prepare target: %w", err)
```

Comment:
please use `github.com/go-errors/errors` instead of fmt.Errorf for better consistency

Reply:
Since go 1.13 `go-errors` is not necessary as `fmt.Errorf` and `errors` take care of things that `go-errors` covered https://go.dev/blog/go1.13-errors.

Actually Unwrap works better with `fmt.Errorf` for reference https://go.dev/play/p/U896OQVWlW4